src/core/data-cite.js

```diff
@@ -52,11 +54,18 @@ function requestLookup(conf) {
     }
     switch (elem.localName) {
       case "a": {
+        if (elem.text === "") {
+          elem.text = title;
+        }
         elem.href = href;
         break;
       }
       case "dfn": {
-        wrapInner(elem, hyperHTML`<a href="${href}"></a>`);
+        if (elem.textContent === "") {
```

Comment:
This still looks weird to me because the two `wrap`s are basically doing the same thing... I was expecting maybe just (no `if` statement at all):

```
wrapInner(elem, hyperHTML`<a href="${href}">${title}</a>`);
```

Because title should just be empty as per line 28, right?

Reply:
Ok, yes. I guess what’s mostly bothering me here is the code repetition, so maybe:

```
const wraper = hyperHTML`<a href="${href}">${
  elem.textContent === “” ?  title : “”
}</a>`);
```

(Formatted correctly... on my phone 📱😅)

Reply:
Done. :)